library/asb_encode_binding.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
```

Comment:
Chatted with @cfchase and we decided that at the current complexity of the module this was more trouble than it was worth.